Output channel automatically focused

Some years ago #57 was reported and fixed, but I'm still getting the same behavior if `node` is not found and it is _very_ annoying.

I use `direnv` + `nix` for handling project dependencies and sometimes the environment is not loaded.

This is what I get in the output channel:
```
[ERROR] Problem launching node_modules/.bin/dprint. node_modules/.bin/dprint: line 20: exec: node: not found

[ERROR] Error initializing dprint. Ensure it is globally installed on the path (see https://dprint.dev/install) or specify a "dprint.path" setting to the executable.
[INFO] Configuration file not found.
[INFO] Extension active!
```

My suggestion is using a notification like other extensions; is not invasive and doesn't change the user workspace but still provides useful information!

--- a/src/legacy/FolderService.ts
+++ b/src/legacy/FolderService.ts
@@ -66,7 +66,8 @@ export class FolderService implements vscode.DocumentFormattingEditProvider {
     const isInstalled = await dprintExe.checkInstalled();
     this.#assertNotDisposed();
     if (!isInstalled) {
-      this.#logErrorAndMaybeFocus(
+      this.#logErrorAndMaybeNotify(
+        "Failed initializing dprint.",
         `Error initializing dprint. Ensure it is globally installed on the path (see https://dprint.dev/install) `
           + `or specify a "dprint.path" setting to the executable.`,
       );
@@ -99,7 +100,11 @@ export class FolderService implements vscode.DocumentFormattingEditProvider {
         throw err;
       }
 
-      this.#logErrorAndMaybeFocus(`Error initializing in ${dprintExe.initializationFolderUri.fsPath}:`, err);
+      this.#logErrorAndMaybeNotify(
+        "Failed initializing dprint.",
+        `Error initializing in ${dprintExe.initializationFolderUri.fsPath}:`,
+        err,
+      );
       return false;
     }
   }
@@ -168,12 +173,12 @@ export class FolderService implements vscode.DocumentFormattingEditProvider {
     return getDprintConfig(this.uri);
   }
 
-  #logErrorAndMaybeFocus(message: string, ...args: any[]) {
+  #logErrorAndMaybeNotify(notificationMessage: string, message: string, ...args: any[]) {
     if (this.#configUri == null) {
-      // only log... don't annoy people with focusing in this case
+      // only log... don't annoy people with notifications in this case
       this.#logger.logError(message, ...args);
     } else {
-      this.#logger.logErrorAndFocus(message, ...args);
+      this.#logger.logErrorAndNotify(notificationMessage, message, ...args);
     }
   }
 }

--- a/src/logger.ts
+++ b/src/logger.ts
@@ -61,7 +61,7 @@ export class Logger {
     // only notify max one time per session to not annoy people
     if (!Logger.#hasFocused) {
       Logger.#hasFocused = true;
-      const buttonText = "Go to Output";
+      const buttonText = "Go to output";
       vscode.window.showWarningMessage(notificationMessage, buttonText).then(selection => {
         if (selection === buttonText) {
           this.#outputChannel.show();

--- a/src/logger.ts
+++ b/src/logger.ts
@@ -1,4 +1,4 @@
-import type * as vscode from "vscode";
+import * as vscode from "vscode";
 
 export class Instant {
   #time: number;
@@ -56,12 +56,17 @@ export class Logger {
     this.#outputChannel.appendLine(getFormattedMessageWithLevel("error", message, args));
   }
 
-  logErrorAndFocus(message: string, ...args: any[]) {
+  logErrorAndNotify(notificationMessage: string, message: string, ...args: any[]) {
     this.logError(message, ...args);
-    // only focus max one time per session to not annoy people
+    // only notify max one time per session to not annoy people
     if (!Logger.#hasFocused) {
       Logger.#hasFocused = true;
-      this.#outputChannel.show();
+      const buttonText = "Go to Output";
+      vscode.window.showWarningMessage(notificationMessage, buttonText).then(selection => {
+        if (selection === buttonText) {
+          this.#outputChannel.show();
+        }
+      });
     }
   }
 }

